### PR TITLE
Added the missing map-component to layout.schema.v1.json

### PR DESCRIPTION
**Pull request by framitdavid**
## Related Issue(s)

[Kartkomponent for stedfestelse i kart](https://github.com/Altinn/altinn-studio/issues/8480)

## Verification

- Manual testing
  - [ ] I have tested these changes manually
  - [ ] Creator of the original issue (or service owner) has been contacted for manual testing
  - [ ] No testing done/necessary
- Automated tests
  - [ ] Unit test(s) have been added
  - [ ] Cypress E2E test(s) have been added
  - [ ] No automatic tests are needed here
  - [ ] I want someone to help me make some tests
- User documentation @ [altinn-studio-docs](https://github.com/Altinn/altinn-studio-docs)
  - [ ] Has been updated
  
  - [ ] No changes/updates needed
- Changes/additions to component properties
  - [ ] Changes are reflected in both `src/layout/layout.d.ts` and `layout.schema.v1.json`, and these are all backwards-compatible
  - [ ] No changes made
- Support in Altinn Studio
  - [ ] Issue(s) created for support in Studio
  
  - [ ] This change/feature does not require any changes to Altinn Studio
- Sprint board
  - [ ] The original issue (or this PR itself) has been added to the Team Apps project and to the current sprint board

### CODE REVIEW

File: schemas/json/layout/layout.schema.v1.json [hunk 1]
```diff
@@ -323,6 +324,54 @@
...
+        "layers": {
+          "type": "object",
+          "title": "Layers",
+          "description": "Map layer",
+          "required": ["url"],
+          "properties": {
+            "url": {
+              "type": "string",
+              "title": "Map layer url",
+              "description": "Url to the map layer"
```
Comment (olemartinorg) [inline schemas/json/layout/layout.schema.v1.json:338]:
```suggestion
              "description": "Url to a map tile. {z}/{x}/{y} will be replaced with tile coordinates, {s} will be replaced with a random subdomain if subdomains are given"
```

File: schemas/json/layout/layout.schema.v1.json [hunk 2]
```diff
@@ -323,6 +324,54 @@
...
+            },
+            "attribution": {
+              "type": "string",
+              "title": "Attribution",
+              "description": "Ascribing a work or remark to a particular unit for recognition"
+            },
+            "subdomains": {
+              "type": "array",
+              "title": "Subdomains"
```
Comment (olemartinorg) [inline schemas/json/layout/layout.schema.v1.json:347]:
```suggestion
              "title": "List of subdomains. Used for balancing the load on different map tiling servers. A random one will replace {s} in the defined url."
```

Comment (olemartinorg) [reply schemas/json/layout/layout.schema.v1.json:347]: We could probably also have `"items": { "type": "string" }` in here as well, to indicate each array item is supposed to be a string.

**Review by olemartinorg — APPROVED** [review]
Great work! Adding some things you could look into if you want, but this is just fine as it is! :1st_place_medal:

File: schemas/json/layout/layout.schema.v1.json [hunk 3]
```diff
@@ -323,6 +324,54 @@
         }
       }
     },
+    "map": {
```
[the remark this answers is not in the document]
Comment (framitdavid) [reply schemas/json/layout/layout.schema.v1.json:327]: Great suggestion, I agree!

File: schemas/json/layout/layout.schema.v1.json (hunk 2 above)
Comment (framitdavid) [reply schemas/json/layout/layout.schema.v1.json:347]: Great feedback, i keep the title to be Subdomains only but added a description with _"List of subdomains. Used for balancing the load on different map tiling servers. A random one will replace {s} in the defined url."_.

Comment (olemartinorg) [reply schemas/json/layout/layout.schema.v1.json]: Ah, yes, I agree! I didn't notice I put everything in the title.. ;-)

**Review by olemartinorg — APPROVED** [review]

**Pull request merged**